perf/pinot/build.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0"?>
+<!--
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+<project name="Pinot" default="build" basedir=".">
+	<taskdef resource="net/sf/antcontrib/antlib.xml" />
+	<description>Pinot</description>
+
+	<!-- set global properties for this build -->
+	<property name="DEST" value="${BUILD_ROOT}/perf/Pinot" />
+	<property name="SRC" location="." />
+
+	<target name="init">
+		<mkdir dir="${DEST}" />
+	</target>
+
+	<target name="getMaven"  description="Download Maven">
+		<var name="MVN_FILE" value="apache-maven-3.8.8-bin.zip"/>
+		<if>
+			<available file="${MVN_FILE}" type="file" />
+			<then>
+				<echo message="${MVN_FILE} exists. Hence, not downloading it." />
+			</then>
+			<else>
+				<echo message="Downloading Maven"/>
+				<var name="mvn_url" value="https://dlcdn.apache.org/maven/maven-3/3.8.8/binaries/apache-maven-3.8.8-bin.zip"/>
+				<echo message="wget ${mvn_url}" />
+				<exec executable="wget" failonerror="true" dir=".">
+					<arg line="${mvn_url}" />
+				</exec>
+			</else>
+		</if>
+		<unzip src="${MVN_FILE}" dest="${BUILD_ROOT}/perf/" />
+	</target>
+
+	<target name="getPinot" depends="init" description="Clone the distribution">
+		<var name="DIR_NAME" value="pinot"/>
+		<if>
+			<available file="${DIR_NAME}" type="dir" />
+			<then>
+				<echo message="${DIR_NAME} exists. Hence, not cloning it." />
+			</then>
+			<else>
+				<echo message="Cloning Pinot"/>
+				<var name="git_command" value="clone --depth 1 -b master https://github.com/apache/pinot.git"/>

```

Comment:
I think we might want to Clone the repository at an specific SHA, otherwise every time we run the benchmark we might be running a different code and thus making comparisons of results of different runs impractical.

Reply:
done